tests/lib/openstack_query/test_utils.py

```diff
@@ -0,0 +1,77 @@
+import unittest
+from datetime import datetime
+
+from unittest.mock import patch
+from parameterized import parameterized
+from nose.tools import raises
+
+import openstack_query.utils as utils
+from exceptions.missing_mandatory_param_error import MissingMandatoryParamError
+
+
+class UtilsTests(unittest.TestCase):
+    """
+    Runs various tests to ensure that utils module functions expectedly
+    """
```

Comment:
Were missing a test for `currentTime`, this can simply be a patch to check we forward on correctly

I want to keep currentTime as we can simply do this in other tests
```
mocked = autospec(TimeUtils)
# ...
mocked.current_time.assert_called_once()
```

Reply:
do we really need a test for this - I can't simply patch it - I have to use `freezegun` package
https://stackoverflow.com/questions/13073281/how-to-mock-pythons-datetime-now-in-a-class-method-for-unit-testing

(that's why I made that method to begin with)